### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by joe-boadi**
Code refactored to meet the requirements

### CODE REVIEW

File: arrayTransformation.js
Comment (E-Agbettor) [inline arrayTransformation.js:1]: Kindly, remove console.log() code in your files

Comment (E-Agbettor) [reply arrayTransformation.js:1]: Also, use commenting like this example

```
/**
 * Calculates the area of a rectangle
 * @param {number} length - The length of the rectangle
 * @param {number} width - The width of the rectangle
 * @returns {number} - The area of the rectangle
 */
function calculateArea(length, width) {
    return length * width;
}
```

**Review by E-Agbettor — CHANGES_REQUESTED** [review]

File: arrayTransformation.js
Comment (joe-boadi) [reply arrayTransformation.js:1]: alright, well noted.

**Review by E-Agbettor — COMMENTED** [review]
Well done!

**Pull request merged**